scripts/cray_runall.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+alps_present=`which alps`
+if [ $? == 0 ]; then
+launcher="aprun -n 1"
+else
+launcher="srun -n 1"
+fi
+echo Running fi_info
+$launcher ./fi_info 2>&1 | tee test.output
```

Comment:
Hmm.. but what about tests in other directories, e.g., *ported*?  Would we want a script for each directory?